govcd/api_vcd.go

```diff
@@ -92,6 +92,14 @@ func (vcdClient *VCDClient) vcdCloudApiAuthorize(user, pass, org string) (*http.
 	}
 
 	defer resp.Body.Close()
+
+	// Catch HTTP 401 (Status Unauthorized) to return an error as otherwise this library will return
+	// odd errors while doing lookup of resources and confused user.
```

Comment:
```suggestion
	// Catch HTTP 401 (Status Unauthorized) to return an error as otherwise this library would return
	// odd errors while doing lookup of resources and confuse user.
```